### ISSUE TITLE
[website] The code blocks' line heights are too small

### ISSUE THREAD

**Issue by ArtskydJ** [sdmp/sdmp.github.io#27]
When viewing http://sdmp.io in chrome, the bottom of lines look cut off:
![sdmp-line-height](https://cloud.githubusercontent.com/assets/1833684/12153300/deca6ebe-b47e-11e5-856e-5f808dd817c5.PNG)
(In case you don't see what I'm saying; look at the `p` in `sdmp`, and the `[` after `schemas`

I am able to fix this by bumping `line-height`:
![sdmp-line-height-2](https://cloud.githubusercontent.com/assets/1833684/12153393/4004a2c6-b47f-11e5-8fa4-a7fc249dd8bd.PNG)

**Comment by saibotsivad** [comment]
Awesome, thanks for noticing that!

**Comment by saibotsivad** [comment]
I'm looking at [this section](http://sdmp.io/core/container/#example) in Chrome 47.0.2526.106 (64-bit) (for Mac) and I'm not seeing anything funny. I also tried it in Safari and Firefox and it displayed correctly. I don't have a Windows machine, so I can't test it on those, but it also displays correctly on all Mac browsers if I set `line-height: 1.4;` instead of `line-height: 1.4em;` which it is currently.
